pyproject.toml

```
@@ -35,16 +35,16 @@ classifiers = [
 requires-python = ">= 3.11, < 4"
 dependencies = [
   "python-dateutil >= 2.8.2, < 3.0",
-  "typing-extensions == 4.11.0",
+  "typing-extensions == 4.12.0",
   # Make sure to update the version for cross-referencing also in the
   # mkdocs.yml file when changing the version here (look for the config key
   # plugins.mkdocstrings.handlers.python.import)
   "frequenz-sdk == v1.0.0-rc6",
-  "frequenz-channels == 1.0.0",
+  "frequenz-channels == 1.0.1",
```

This should be widened:
```suggestion
  "frequenz-channels => 1.0.1, < 2.0.0",
```